pkgs/top-level/all-packages.nix

```diff
@@ -20173,8 +20158,6 @@ with pkgs;
   swig = swig3;
   swigWithJava = swig;
 
-  c2ffi = callPackage ../development/tools/misc/c2ffi { };
```

Comment:
Gentoo has a version from 2023 — would it be easy enough to just update this rather than dropping it?